src/tests/components/socialLogin/facebook.test.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import FacebookOption from '../../../components/social/facebook';
+
+// Define a shared shallow renderer function
+
+const facebookButton = (props = {}) => {
+  const facebook = shallow(<FacebookOption {...props} />);
+  return facebook;
+};
+
+describe('Facebook Login', () => {
+  // Pass the value of the shallow renderer before each test
+  let facebook;
+  beforeEach(() => {
+    facebook = facebookButton();
+  });
+
+  it('matches the snapshot', () => {
+    const FacebookButton = shallow(<FacebookOption />);
+    expect(FacebookButton).toMatchSnapshot();
+  });
+  it('should render without errors', () => {
+    const wrapper = shallow(<FacebookOption />);
+    expect(wrapper).toMatchSnapshot();
+  });
+
+  it('should work as expected', () => {
```

Comment:
What are you testing for here?

Reply:
@Yiga-fred I have rewritten the description to suit the test